test/e2e/accounts/common.ts

```diff
@@ -373,8 +373,11 @@ export async function createBtcAccount(driver: Driver) {
     text: messages.addNewBitcoinAccount.message,
     tag: 'button',
   });
-  await driver.clickElementAndWaitToDisappear({
-    text: 'Create',
-    tag: 'button',
-  });
+  await driver.clickElementAndWaitToDisappear(
+    {
+      text: 'Create',
+      tag: 'button',
+    },
+    5000,
```

Comment:
**Note:** we suspect that the test was failing because there was a time-out while waiting for the "account name" dialog to disapear, this should improve the stability of the test.

Reply:
Implemented in: [278dba6](https://github.com/MetaMask/metamask-extension/pull/26082/commits/278dba6ac79a94b0c9fcf7175e593dcba8b4adde), thanks!